playbooks/roles/apache2/tasks/main.yaml

```diff
@@ -31,7 +31,8 @@
 
 - name: enable mpm_prefork
   command: a2enmod mpm_prefork
-  when: not mpm_prefork.stat.exists
+  when: not mpm_prefork.stat.exists and
+      (ansible_lsb.id == "Ubuntu" and ansible_lsb.release == "14.04")
```

Comment:
apparently `mpm_prefork` is the default for apache 2.2 

http://httpd.apache.org/docs/2.2/mpm.html#defaults